src/rust/ensogl/src/display/shape/text/text_field/content/line.rs

```diff
@@ -99,11 +98,16 @@ pub struct LineFullInfo<'a> {
 }
 
 impl<'a> LineFullInfo<'a> {
+    /// Gets line y position.
```

Comment:
Please add info if it is a middle of the line, a _baseline_ or maybe a top of the line area (as it seem to be from `baseline_start` function.